metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -61,7 +63,8 @@ final case class InitializationOptions(
     statusBarProvider: Option[String],
     treeViewProvider: Option[Boolean],
     openNewWindowProvider: Option[Boolean],
-    copyWorksheetOutputProvider: Option[Boolean]
+    copyWorksheetOutputProvider: Option[Boolean],
+    stripDAPOutputColor: Option[Boolean]
```

Comment:
```suggestion
    disableColorOutput: Option[Boolean]
```
does a more general flag make sense?

Reply:
Yea I went back and forth about whether it should include DAP or not. I went ahead and made it more generic. 👍🏼